modules/quickstart/pages/network-quickstart.adoc

```diff
@@ -122,12 +122,12 @@ cd hello
 ----
 
 [[ping-the-network]]
-== Check the connection to the network
+== Check the connection to the {IC} blockchain
 
-Before you can deploy your first project on the {IC}, you need to connect to the network gateway running remotely on a subnet that you can access.
-There is a reserved network alias that you can use to access the {IC} running remotely on nodes in data centers. The network alias is a system setting that's defined internally, so there's nothing you need to configure in your projects by default.
+Before you can deploy your first project on the {IC} blockchain, you need to connect to the network gateway running remotely on a subnet that you can access.
+There is a reserved network alias that you can use to access the {IC} blockchain. The network alias is a system setting that's defined internally, so there's nothing you need to configure in your projects by default.
 
-To check your connection to the {IC} network:
+To check your connection to the {IC} blockchain:
 
 [arabic]
 . Check that you are in the root directory for your project, if needed.
```

Comment:
Maybe in the line below (I cannot comment) we can change "network" with "mainnet". But to me this is also fine